src/datascience-ui/interactive-common/variableExplorer.tsx

```diff
@@ -212,7 +211,7 @@ export class VariableExplorer extends React.Component<IVariableExplorerProps, IV
         return (
             <div id="variable-panel" ref={this.variablePanelRef}>
                 <div id="variable-panel-padding">
-                    <div className="variable-explorer" ref={this.variableExplorerRef}>
+                    <div className="variable-explorer" ref={this.variableExplorerRef} style={variableExplorerStyles}>
```

Comment:
Suppressing them all actually turned up one issue here, the height sizing didn't really matter (as the grid sizing took care of this) but the font was incorrect.